doc/modules/qualitymetrics/amplitude_cutoff.rst

```diff
@@ -6,15 +6,16 @@ Amplitude cutoff (:code:`amplitude_cutoff`)
 Calculation
 -----------
 
-A histogram of spike amplitudes is created and deviations from the expected symmetrical distribution are identified.
+First, all spike amplitudes for one unit are collapsed across time to create a histogram, then the histogram is smoothed using a 1D Gaussian filter. For some units, the amplitude histogram will not fall off gracefully to zero at the lower end, indicating that some spikes were likely missed by the sorter. To calculate the approximate fraction of missing spikes, we look at the height of the lowest amplitude bin, and count the number of spikes at the upper end of the distribution with amplitudes above a bin of similar height. The ratio of this count to the total number of spikes yields the amplitude cutoff.
 
 Expectation and use
 -------------------
 
-Deviations from the expected Gaussian distribution are used to estimate the number of spikes missing from the unit.
-This yields an estimate of the number of spikes missing from the unit (false negative rate).
-A smaller value for this metric is preferred, as this indicates fewer false negatives.
-The distribution can be computed on chunks for larger recording, as drift can impact the spike amplitudes (and thus not give a Gaussian distribution anymore).
+This metric can be considered an estimate of the fraction of false negatives (missed spikes) during the intervals over which a unit is successfully detected. Therefore, smaller amplitude cutoff values tend to indicate higher quality units. However, the actual fraction of missed spikes may be much higher if the unit is not tracked for a portion of the recording session. For this reason, amplitude cutoff should be combined with a metric such as presence ratio to more accurately estimate each unit's overall completeness.
+
+The calculation works best when the amplitude distribution is *symmetric* and has a *single peak*. Importantly, these assumptions likely do not hold if there is substantial drift. In this case, amplitude cutoff can be computed separately for different chunks of the recording, yielding a more accurate estimate of the fraction of spikes missing from each chunk.
+
+**IMPORTANT:** When applying this metric, it is critical to know how the spike amplitudes were calculated. Template-based sorting algorithms such as Kilosort output the template scaling factors that are used to fit particular templates to each spike waveform. By multiplying the template amplitude by the scaling factor, one can approximate the original amplitude of each spike. However, this amplitude often does not match the true amplitude. Amplitude cutoffs computed from the template scaling factors (`amplitudes.npy` in the Kilosort output) tend to be much higher than when using actual spike amplitudes extracted from the raw data. It's important to know which method was used in order to compare amplitude cutoff values across experiments.
```

Comment:
I guess my only question here is that spikeinterface isn't using `amplitudes.npy` so do we want to mention that here?

Reply:
Yep, just added a sentence about that!

Reply:
Thanks. That looks good to me!